lib/src/lib/models/form-field-base.ts

```diff
@@ -52,4 +52,5 @@ export interface FormFieldBaseOptions {
   infoTooltip?:
     | { text: string; icon?: string; class?: string }
     | ((data?: any) => { text: string; icon?: string; class?: string } | null);
+  customId?: string;
```

Comment:
ik zou dit `elementId` noemen, met als comment dat dit default de attribute name is.